src/accumulator/stump.rs

```diff
@@ -167,7 +193,36 @@ mod test {
         assert!(s.leafs == 0);
         assert!(s.roots.len() == 0);
     }
+    #[test]
+    fn test_update_data_add() {
+        let preimages = vec![0, 1, 2, 3];
+        let hashes = preimages
+            .into_iter()
+            .map(|preimage| hash_from_u8(preimage))
+            .collect();
+
+        let (_, updated) = Stump::new()
+            .modify(&hashes, &vec![], &Proof::default())
+            .unwrap();
 
+        let positions = vec![0, 1, 2, 3, 4, 5];
+
+        let hashes: Vec<_> = vec![
+            "6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d",
+            "4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a",
+            "dbc1b4c900ffe48d575b5da5c638040125f65db0fe3e24494b76ea986457d986",
+            "084fed08b978af4d7d196a7446a86b58009e636b611db16211b65a9aadff29c5",
+            "02242b37d8e851f1e86f46790298c7097df06893d6226b7c1453c213e91717de",
+            "9576f4ade6e9bc3a6458b506ce3e4e890df29cb14cb5d3d887672aef55647a2b",
```

Comment:
Just leaving a note:

The go library currently also returns the roots as well so if we don't return them here, the behaviors won't be the same.

Not sure if returning the root or not returning the root is better.

Reply:
Do you mean the current set of roots?

Reply:
Not the current set of roots. I mean the roots that were generated by the additions.

So here, the final root after adding all 6 elements was `df46b17be5f66f0750a4b3efa26d4679db170a72d41eb56c3e4ff75a58c65386`.

In the go implementation, this root would also be returned.